tag_manager/authorized_api/src/tags/schema/tag_templates/ab_tasty_attributes.json

```diff
@@ -11,7 +11,7 @@
             ]
         },
         "template_options": {
-            "description": "Tag template options",
+            "description": "[Tag template options](#tag-template-options)",
```

Comment:
Can you check on preview that these links work as intended? I'm not sure they do.
![2021-04-27T154702](https://user-images.githubusercontent.com/4488774/116252719-1f67bc80-a770-11eb-8e25-104f2865ceaf.png)

Reply:
The preview works ok on my end. Any specific tag is selected (maybe dome templates were missed)?

Reply:
There is an issue with empty template options. I am fixing this now.

Reply:
Fixed